.github/workflows/docs.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Upload docs artifact
         uses: actions/upload-pages-artifact@v1
         with:
-          path: 'build/docs'
+          path: 'docs/build'
```

Comment:
Please verify that this actually also works on your computer and indeed provides a speed up. Note that the speedup will not be visible (yet) when using the `tox` based command (plan to do this in an upcoming PR).

Reply:
Which command exactly do you want us to run?

Reply:
`sphinx-build -b html docs docs/build -n -W`

Reply:
Think we are not talking about the same thing. You asked to run the command to see if it now (on your PR) works faster then before. But when I run the command on the commit where your PR branches of (that is, on the respective commit of the main branch where you started working), I get the above error. Hence, I cannot compare, because I don't get the timing for "previous version"

Reply:
Ahhhhh yes you are perfectly right. I was not being precise in what I wanted you to test 😅 
So here is my request: Run that command that I wrote here after having built the documentation once via `tox` on the tip of the branch and just report the time. It should be significantly faster than the `tox` run and properly include your local changes.

Reply:
Well ... yes ... but isn't the answer trivial here? Because essentially what you ask us to to here is: "run a task A and time it, and then run a task B **that entirely contains task A as a subtask** and see if it takes longer" 😄 You see what I mean?

So yes, it runs fast, but technically speaking that doesn't necessarily mean that it wouldn't have been fast before the PR.

Reply:
I meant the corresponding subtask within `tox` itself. I tried to explain that in our dev meeting but seem to have failed, so here is now the full explanation:

- Our `tox` actually performs two calls: One for Linkchecking, one for Building. The linkchecking call checks the links (duh), but also already does some preparation work for the full build. This second built is exactly the `sphinx-build....` stuff that we are talking about here.
- Now, what was the state so far? So far, we built the documentation (via the full `tox` command) and moved it to another folder. This made it *impossible* for sphinx to "smartly" build the documentation: That is, even if the documentation already existed, it was not possible to make any use out of the fact that we already have a built documentation. This is true for both using the `tox` command and just the "second part" of that, i.e., `sphinx-build....`. That is, this second call **would have built the full documentation from scratch again**.
- This is now different: With the change of location, the `sphinx-build....` command realizes that there is stuff already built and does not do everything from scratch. This means that if there is an already built documentation, running the `sphinx-build ...` command will be able to see that there is already some stuff present and does not need to to everything from scratch. This should result in a more reasonable time of building the documentation by that command. It should only take 1-2 minutes (maybe 3, don't remember the time exactly)
- The comparison I wanted to make is that this is indeed faster than running the `tox` command, since this is the currently intended way of building the documentation.

Is it now clearer? 😄  In any case, a small "Tested, works, is faster" is also fine 😉